ls1intum/Artemis
# Display active system notifications for unauthenticated users

### Checklist

### Motivation and Context
As system notifications (and notifications in general) are distributed via websockets that are bound to an authenticated user these notifications are not displayed for unauthenticated users. However, they are often as relevant for unauthenticated as for authenticated users (e.g. information about current downtime).

### Description
- Made api/system-notifications/active-notification REST route accessible to unauthenticated users
- Added explicit request for active system notifications if user is not authenticated

### Steps for Testing
1. Log in to Artemis
2. Navigate to *Server Administration* > *Notification Management*
3. Create an active system notification
4. Log out and reload the page
5. The created system notification should be displayed on the login page

### Screenshot
<img width="869" alt="Bildschirmfoto 2019-12-10 um 09 27 40" src="https://user-images.githubusercontent.com/15168372/70511703-ec4b6b80-1b2f-11ea-8f6e-545500a3aa5d.png">
